MODULE.bazel

```diff
@@ -109,15 +117,12 @@ git_override(
 git_override(
     module_name = "gazelle",
     commit = "4d0129d7adae33e8e648d3624aff8fa8f93f375b",
-    patches = [
-        "//:patches/gazelle/dont-flatten-srcs.diff",
-        "//:patches/gazelle/googleapis.diff",
```

Comment:
Please delete the file `patches/gazelle/googleapis.diff` from the repository.

Reply:
Done